main.go

```diff
@@ -34,6 +35,11 @@ func main() {
 		}()
 	}
 
+	go func() {
```

Comment:
There's already `cfg.ListenPprof` -> maybe we can use that instead of another hardcoded HTTP port? Maybe we can rename `cfg.ListenPprof` to something more general, like `ListenAux` or whatever. Or add additional config variable `ListenMetrics` and use that (with same default value of `:9090`)